source/features/parse-backticks.tsx

```diff
@@ -1,41 +1,22 @@
 import './parse-backticks.css';
 import onetime from 'onetime';
 import {observe} from 'selector-observer';
-import zipTextNodes from 'zip-text-nodes';
 
 import features from '.';
 import {parseBackticks} from '../github-helpers/dom-formatters';
-import parseBackticksCore from '../github-helpers/parse-backticks';
 
 function init(): void {
 	const selectors = [
 		'.BorderGrid--spacious .f4.mt-3', // `isRepoHome` repository description
-		'.js-commits-list-item .mb-1', // `isCommitList` commit message
-		'.js-commits-list-item pre', // `isCommitList` commit description
-		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot` commit message (GHE #4021)
-		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.Link--primary', // `isRepoRoot` commit message
-		'.commit-title', // `isCommit` commit message
-		'.commit-desc', // `isCommit` commit description
-		'.js-commit .pr-1 > code', // `isPRConversation` pushed commits
-		'.js-details-container .pr-1 > code', // `isCompare` pushed commits
-		'.Box-row .mb-1 a', // `isCompare` open Pull Request title
-		'.blame-commit-message', // `isBlame` commit message
-		'[id^=ref-issue-]', // `isIssue` issue and PR references
-		'[id^=ref-pullrequest-]', // `isPRConversation` issue and PR references
-		'[aria-label="Link issues"] a', // `isIssue`, `isPRConversation` linked issue and PR
-		'.Box-header.Details .link-gray', // `isSingleFile` commit message (GHE #4021)
-		'.Box-header.Details .Link--secondary', // `isSingleFile` commit message
-		'.Box-header.Details pre', // `isSingleFile` commit description
 		'.release-header', // `isReleasesOrTags` Headers
-		'.existing-pull-contents .list-group-item-link', // `isCompare` with existing PR
+		'.Box-row .mb-1 a', // `isCompare` with existing PR
 		'#pull-requests a.link-gray-dark', // `isPulse` issue and PR title (GHE #4021)
 		'#pull-requests a.Link--primary', // `isPulse` issue and PR title
 		'[id^="check_suite"] a.link-gray-dark', // `isRepositoryActions` (GHE #4021)
 		'[id^="check_suite"] a.Link--primary', // `isRepositoryActions`
-		'.checks-summary-conclusion + .flex-auto .f3', // `isActions` run
+		'.js-socket-channel[data-url*="/header_partial"] h3', // `isActions` run
 		'.js-wiki-sidebar-toggle-display a', // `isWiki` sidebar pages title
-		'.wiki-wrapper .gh-header-title', // `isWiki` page title
-		'.js-recent-activity-container .text-bold', // `isDashboard` "Recent activity" titles
+		'#wiki-wrapper .gh-header-title', // `isWiki` page title
 		'.issues_labeled .text-gray-dark > a', // `isDashboard` "help wanted" event titles (GHE #4021)
 		'.issues_labeled .color-text-primary > a', // `isDashboard` "help wanted" event titles
 		'.commits blockquote', // `isDashboard` newsfeed commits
```

Comment:
I can't test this, but I think it should be removed a priori since they're hidden by `hide-noisy-newsfeed-events`

Reply:
Indeed, since `hide-noisy-newsfeed-events` can't be disabled this is parsing backticks people will never see.

Reply:
> Indeed, since [`hide-noisy-newsfeed-events`](/sindresorhus/refined-github/blob/main/source/features/hide-noisy-newsfeed-events.tsx) can't be disabled

It can be disabled, it's a "CSS-only" JS feature 😃